app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ActiveRecord::Base
+  before_create :generate_token
```

Comment:
This is actually a huge security threat, whoever has access to the database could take control over any account. This should be generated upon request from user.